gno.land/pkg/gnoweb/alias.go

```diff
@@ -55,3 +57,8 @@ func AliasAndRedirectMiddleware(next http.Handler, analytics bool) http.Handler
 		next.ServeHTTP(w, r)
 	})
 }
+
+// IsHomePath checks if the given path is the home path.
+func IsHomePath(path string) bool {
+	return path == "/" || path == HomeAlias
```

Comment:
just `path == "/"`

if we go to `/r/gnoland/home`, i expect to see the full UI.